src/api/uploads/FolderUploadNode.js

```diff
@@ -231,6 +265,14 @@ class FolderUploadNode {
             this.readEntry(reader, resolve);
         });
     };
+
+    /**
+     * Returns the folderId
+     * @returns {string}
```

Comment:
will `folderId` be `''` or some value, or can this ever end up as `undefined`? if the latter, we'd want a maybe return type or handle that case somehow

Reply:
The empty state is an empty string -> it is returned from the monolith that way.